Source/Charts/Renderers/XAxisRenderer.swift

```diff
@@ -339,10 +329,7 @@ open class XAxisRenderer: AxisRendererBase
     
     @objc open func drawGridLine(context: CGContext, x: CGFloat, y: CGFloat)
     {
-        guard
-            let viewPortHandler = self.viewPortHandler
-            else { return }
-        
+
```

Comment:
empty line? Not sure if this is github display issue, but if old guard is deleted, no space line needed. (ignore if there is no empty line)

Reply:
I'll fix